### ISSUE TITLE
Ngx-schema-form 2.3.4 causes Angular 8 build to fail

### ISSUE THREAD

**Issue by laxu** [guillotinaweb/ngx-schema-form#291]
With Ngx-schema-form 2.3.4 Angular 8 `ng build` seems to fail. You can replicate this by creating a new Angular app and adding ngx-schema-form to it. It gives errors like this:

```ERROR in ./node_modules/ngx-schema-form/fesm5/ngx-schema-form.js 5364:24-30
"export 'default' (imported as 'common') was not found in '@angular/common'
ERROR in ./node_modules/ngx-schema-form/fesm5/ngx-schema-form.js 4551:30-36
"export 'default' (imported as 'common') was not found in '@angular/common'
ERROR in ./node_modules/ngx-schema-form/fesm5/ngx-schema-form.js 5362:16-20
"export 'default' (imported as 'core') was not found in '@angular/core'
ERROR in ./node_modules/ngx-schema-form/fesm5/ngx-schema-form.js 5335:31-35
"export 'default' (imported as 'core') was not found in '@angular/core'
ERROR in ./node_modules/ngx-schema-form/fesm5/ngx-schema-form.js 5334:30-34
"export 'default' (imported as 'core') was not found in '@angular/core'
ERROR in ./node_modules/ngx-schema-form/fesm5/ngx-schema-form.js 5318:16-20
"export 'default' (imported as 'core') was not found in '@angular/core'
ERROR in ./node_modules/ngx-schema-form/fesm5/ngx-schema-form.js 5294:24-28
"export 'default' (imported as 'core') was not found in '@angular/core'
ERROR in ./node_modules/ngx-schema-form/fesm5/ngx-schema-form.js 5215:25-29
```

**Comment by ebrehault** [comment]
which version of Angular are you using?

**Comment by laxu** [comment]
Sorry, forgot to add that. Angular 8. Package.json dependencies:

```
"dependencies": {
    "@angular/animations": "~8.1.0",
    "@angular/common": "~8.1.0",
    "@angular/compiler": "~8.1.0",
    "@angular/core": "~8.1.0",
    "@angular/forms": "~8.1.0",
    "@angular/platform-browser": "~8.1.0",
    "@angular/platform-browser-dynamic": "~8.1.0",
    "@angular/router": "~8.1.0",
    "ngx-schema-form": "^2.3.4",
    "rxjs": "~6.4.0",
    "tslib": "^1.9.0",
    "zone.js": "~0.9.1"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.801.0",
    "@angular/cli": "~8.1.0",
    "@angular/compiler-cli": "~8.1.0",
    "@angular/language-service": "~8.1.0",
    "@types/node": "~8.9.4",
    "@types/jasmine": "~3.3.8",
    "@types/jasminewd2": "~2.0.3",
    "codelyzer": "^5.0.0",
    "jasmine-core": "~3.4.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.1.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~2.0.1",
    "karma-jasmine-html-reporter": "^1.4.0",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.15.0",
    "typescript": "~3.4.3"
  }```

**Comment by ebrehault** [comment]
That was due to a bad tsconfig.lib.json setting (see https://github.com/ng-packagr/ng-packagr/issues/1194)

It is fixed now in 2.3.5.
Thank you very much for reporting :)

**Comment by laxu** [comment]
Thank you so much for such a fast response!
